index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<Html>
+<Header>
+  <Title>Melanie Moore Resume</Title>
+</Header>
```

Comment:
Can you please change the "Header" tag to "head". This is technically correct, but I believe the standard is to use the "head" tag for including metadata such as styling and scripts. "Head" tags are also typically seen at the top of the HTML tags. "Header" tags are mainly seen in articles.